shared/src/common/types/groups/group-item-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+type GroupItemResponseDto = {
```

Comment:
```suggestion
type GroupsItemResponseDto = {
```
should be as service name